### PR TITLE
Switch to FilterQuality.medium for images

### PR DESCRIPTION
**Pull request by goderbauer**
https://github.com/flutter/flutter/issues/148253

### CODE REVIEW

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/magnifier.dart [hunk 1]
```diff
@@ -651,7 +651,7 @@ class _RenderMagnification extends RenderProxyBox {
           magnificationScale * ((focalPointOffset.dx * -1) - thisCenter.dx) + thisCenter.dx,
           magnificationScale * ((focalPointOffset.dy * -1) - thisCenter.dy) + thisCenter.dy)
       ..scale(magnificationScale);
-    final ImageFilter filter = ImageFilter.matrix(matrix.storage, filterQuality: FilterQuality.high);
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/widgets/magnifier.dart:654]: I think this one should actually still be high. The other use case for cubic is really, really high scales which the magnifier will do.

Comment (goderbauer) [reply packages/flutter/lib/src/widgets/magnifier.dart]: Reverted back to high.

**Review by eyebrowsoffire — APPROVED** [review]
LGTM!

**Review by chingjun — APPROVED** [review]

**Pull request merged**